v1/index.js

```diff
@@ -12,8 +12,8 @@ const StructuredHTTPReceiver =
 const BinaryHTTPReceiver =
   require("../lib/bindings/http/receiver_binary_1.js");
 
-function event() {
-  return new Cloudevent(Spec);
+function newEvent() {
+  return new CloudEvent(Spec);
 }
```

Comment:
Again - if we are exporting the ctor...

Reply:
@lance If I am correct, spec 0.2 had SDK conformance to expose a method event() to create new event instances. For now, it's just to not break the sdk API

Reply:
I agree. However, this code was here before. I'm just changing the name.